src/cider/nrepl/middleware/debug.clj

```diff
@@ -173,9 +173,9 @@
 (declare read-debug-command)
 
 (defn inspect-then-read-command
-  "Inspect `value` and send it as part of a new `read-debug-command`."
-  [extras value]
-  (let [i (pr-str (:rendered (swap-inspector! *msg* inspect/start value)))]
+  "Inspect `inspect-value` and send it as part of a new `read-debug-command`."
```

Comment:
inspect "inspect-value" - that sounds a bit weird. Maybe if the other 2 params made an appearance in the docstring it would have been better.